stacks/stack_irvot2015.m

```diff
@@ -0,0 +1,38 @@
+function experiments = stack_irvot2015()
+
+set_global_variable('bundle', 'https://liu.box.com/shared/static/4yczc0pb5uwjvvm54zfh601aj5vdz7q7.zip');
+
+baseline.name = 'baseline';
+baseline.converter = [];
+baseline.execution = 'default';
+baseline.labels = {'camera_motion', 'illum_change', 'occlusion', 'size','motion', 'empty'};
+               
+baseline.parameters.repetitions = 15;
+baseline.parameters.burnin = 10;
+baseline.parameters.skip_initialize = 5;
+baseline.parameters.failure_overlap = 0;
+
+region_noise.name = 'region_noise';
```

Comment:
Are you planning to have region noise experiment as well? I am just asking so that this is clear.